src/components/table/table-column-resize/resizable-table-column.component.html

```diff
@@ -1,4 +1,5 @@
-<div class="ux-resizable-table-column-content">
+<div class="ux-resizable-table-column-content"
+    (uxResize)="onResize()">
```

Comment:
Could we use the ResizeService on the table element rather than having to do this on each column? This would setup up a resize observer for every column rather than one per table which could slow performance on larger tables.